### PR TITLE
Create lambda_function.py

### PR DESCRIPTION
**Pull request by n2patil**
Associated code for the Events example for docs.benchling.com: https://docs.benchling.com/docs/example-send-benchling-request-to-a-slack-channel.

### CODE REVIEW

File: events_examples/benchling_request_to_slack/lambda_function.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import json
+import os
+import urllib3
+
+
+def lambda_handler(event, context):
+    http = urllib3.PoolManager()
+    assignee_handles = []
+    for assignee_handles in event["detail"]["request"]["assignees"]:
+        assignee_handles.append("@" + str(assignee_handles["user"]["handle"]))
```
Comment (alexrejto) [inline events_examples/benchling_request_to_slack/lambda_function.py:12]: Can you rename one of the `assignee_handles`? It is a bit confusing that they all have the same name.

**Review by alexrejto — CHANGES_REQUESTED** [review]
Just one style comment!

File: events_examples/benchling_request_to_slack/lambda_function.py (hunk 1 above)
Comment (alexrejto) [inline events_examples/benchling_request_to_slack/lambda_function.py:12]:
```suggestion
    assignee_handles = []
    for assignee in event["detail"]["request"]["assignees"]:
        assignee_handles.append("@" + str(assignee["user"]["handle"]))
```

Comment (n2patil) [reply events_examples/benchling_request_to_slack/lambda_function.py:12]: Good call! Will updated that!

**Review by n2patil — COMMENTED** [review]
updated based on alex's comment

**Review by alexrejto — APPROVED** [review]

**Pull request merged**